.vscode/settings.json

```diff
@@ -18,6 +18,12 @@
 	],
 	"eslint.format.enable": true,
 	"[javascript]": {
-		"editor.defaultFormatter": "dbaeumer.vscode-eslint"
+		"editor.rulers": [80]
 	},
+	"[typescript]": {
+		"editor.rulers": [80]
+	},
+	"[scss]": {
+		"editor.rulers": [80]
+	}
```

Comment:
may i pretty pretty please ask it to be 120, or *at least* 100?
80 is way way too claustrophobic. It's a relict of the 70s and is no longer relevant with modern screen sizes

Reply:
Its just a ruler, this doesn't affect line length in CI checks